Statistics/Stats.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Statistics
+{
+    public class Stats
+    {
+        public Stats(float avg, float max, float min)
+        {
+            Average = avg;
+            Max = max;
+            Min = min;
+        }
+        public float Average { get; set; }
```

Comment:
Good choice of `float` type, given the accuracy specification (`epsilon` of 0.001)